src/main/java/core/db/DBManager.java

```diff
@@ -76,8 +78,6 @@
 
 	private int m_iLatestHRFid = -1;
 
-	private long m_lLatestUpdateTime = -1;
```

Comment:
What was it used for?

Reply:
it was a value derived from hrf_id:

~~~
public void updateLatestData(){
	m_iLatestHRFid = ((HRFTable) getTable(HRFTable.TABLENAME)).getLatestHrf().getHrfId();
	m_lLatestUpdateTime = DBManager.instance().getBasics(m_iLatestHRFid).getDatum().getTime();
}
~~~